src/views/Proposals.vue

```diff
@@ -150,10 +159,40 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['getProposals'])
+    ...mapActions(['getProposals']),
+    checkState() {
+      const located = location.hash.replace(/(^#\/yam\/#)/, "");
```

Comment:
We shouldn't have a namespace hardcoded here, this page must work across all namespaces, it should not be specific to a namespace.

Reply:
for what reason?, on other pages you are already hardcoding namespaces such as loadProposal loadPower..., unless you think to do a separated file for this kind of stuff then we can move it there

Reply:
good catch, updated to be dynamic